File: src/models/Station.php
<?php

class Station implements JsonSerializable
{
  protected $id;
  protected $station_name;
  protected $created;
  public $cams = array();

  public function __get($property)
  {
    if (property_exists($this, $property)) {
      return $this->$property;
    }
  }

  public function __set($property, $value)
  {
    if (property_exists($this, $property)) {
      $this->$property = $value;
    }

    return $this;
  }

  public function jsonSerialize()
  {
    return (object) get_object_vars($this);
  }
}

File: fastapi_app/app/models/station.py
from datetime import datetime

from sqlalchemy import Column, DateTime, String
from sqlalchemy.orm import relationship

from ..db import Base
from .types import unsigned_int


class Station(Base):
    __tablename__ = "station"

    id = Column(unsigned_int(), primary_key=True, autoincrement=True)
    station_name = Column(String(100), unique=True, nullable=False)
    created = Column(DateTime, nullable=False, default=datetime.utcnow)

    cams = relationship("Cam", back_populates="station", cascade="all,delete-orphan")
